src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/PaymentInfo.java

```diff
@@ -6,6 +6,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class PaymentInfo {
 
+    @JsonProperty("document_control_number")
```

Comment:
please explain

Reply:
ok, this model is now also used in functional testst for sending message to the queue (previously only when reading)